sdk/keyvault/keyvault-keys/tsconfig.json

```diff
@@ -1,39 +1,25 @@
 {
   "compilerOptions": {
-    /* Basic Options */
-    "target": "es6" /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */,
-    "module": "es6" /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */,
-    "lib": [] /* lib dependencies are triple-slash directives in lib/index.ts */,
-    "declaration": true /* Generates corresponding '.d.ts' file. */,
-    "declarationMap": true /* Generates a sourcemap for each corresponding '.d.ts' file. */,
-    "sourceMap": true /* Generates corresponding '.map' file. */,
-
+    "alwaysStrict": true,
+    "noImplicitAny": true,
+    "preserveConstEnums": true,
+    "sourceMap": true,
+    "newLine": "LF",
+    "target": "es5",
+    "moduleResolution": "node",
+    "noUnusedLocals": true,
+    "noUnusedParameters": true,
+    "strict": true,
+    "module": "esNext",
     "outDir": "./dist-esm",
-    "stripInternal": true /* Do not emit declarations for code with @internal annotation*/,
-    "declarationDir": "./types/src" /* Output directory for generated declaration files.*/,
-    "importHelpers": true /* Import emit helpers from 'tslib'. */,
-
-    /* Strict Type-Checking Options */
-    "strict": true /* Enable all strict type-checking options. */,
-    "noImplicitReturns": true /* Report error when not all code paths in function return a value. */,
-
-    /* Additional Checks */
-    "noUnusedLocals": true /* Report errors on unused locals. */,
-
-    /* Module Resolution Options */
-    "moduleResolution": "node" /* Specify module resolution strategy: 'node' (Node.js) or 'classic' (TypeScript pre-1.6). */,
-    "allowSyntheticDefaultImports": true /* Allow default imports from modules with no default export. This does not affect code emit, just typechecking. */,
-    "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
-
-    /* Experimental Options */
-    "forceConsistentCasingInFileNames": true,
-
-    /* Other options */
-    "newLine": "LF" /*	Use the specified end of line sequence to be used when emitting files: "crlf" (windows) or "lf" (unix).”*/,
-    "allowJs": false /* Don't allow JavaScript files to be compiled.*/,
-    "resolveJsonModule": true
+    "declaration": true,
+    "declarationMap": true,
+    "importHelpers": true,
+    "declarationDir": "./types/src",
+    "lib": ["dom", "es5", "es6", "es7", "esnext"],
+    "esModuleInterop": true
   },
   "compileOnSave": true,
-  "exclude": ["node_modules", "types/**", "./samples/**/*.ts", "./test/perf/azure-sb-package/*.ts"],
-  "include": ["./src/**/*.ts", "./test/**/*.ts"]
+  "exclude": ["node_modules", "./samples/**/*.ts"],
+  "include": ["./src/**/*.ts", "./tests/**/*.ts"]
```

Comment:
[NIT]
For consistency, name of the tests folder 📂  `tests` -> `test`
servicebus, template and storage packages follow the name "test".

Reply:
This is true! The only caveat is that this PR size will likely increase considerably. Can I change the name of the folder in another PR?